contracts/CoveragePool.sol

```diff
@@ -42,14 +120,11 @@ contract CoveragePool is Ownable {
         external
         onlyApprovedRiskManager
     {
-        uint256 FLOATING_POINT_DIVISOR =
-            CoveragePoolConstants.getFloatingPointDivisor();
-
         uint256 amountToSeize =
             collateralToken
                 .balanceOf(address(assetPool))
                 .mul(portionToSeize)
-                .div(FLOATING_POINT_DIVISOR);
+                .div(CoveragePoolConstants.FLOATING_POINT_DIVISOR);
```

Comment:
Can we do the same - in a separate PR - in all other places where `getFloatingPointDivisor` is used?

Reply:
Sure, I will do that in a separate PR:
https://github.com/keep-network/coverage-pools/pull/54